build-tools/enumification-helpers/generate-const-list-2.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using System.Xml.Linq;
+
+
+public class Driver
+{
+	class Constant {
+		public string Package, ParentType, Level, FieldType, Name, Value;
+		public bool IsTypeInterface;
+	}
+	
+	static string GetApiLevel (string file)
+	{
+		Console.Error.WriteLine ($"[{file}]");
```

Comment:
Do you still need this debug spew?

Reply:
It's not a big deal, we (well most likely only I) use it only when we are going to deal with new API for enumification.